components/admin_console/admin_definition.jsx

```diff
@@ -3408,11 +3429,12 @@ export default {
                     },
                     {
                         type: Constants.SettingsTypes.TYPE_BOOL,
-                        key: 'ServiceSettings.EnableUserAccessTokens',
-                        label: t('admin.service.userAccessTokensTitle'),
-                        label_default: 'Enable User Access Tokens: ',
-                        help_text: t('admin.service.userAccessTokensDescription'),
-                        help_text_default: 'When true, users can create [user access tokens](!https://about.mattermost.com/default-user-access-tokens) for integrations in **Account Settings > Security**. They can be used to authenticate against the API and give full access to the account.\n\n To manage who can create personal access tokens or to search users by token ID, go to the **User Management > Users** page.',
+                        key: 'ServiceSettings.DisableBotsWhenOwnerIsDeactivated',
+                        label: t('admin.service.disableBotOwnerDeactivatedTitle'),
+                        label_default: 'Disable bot accounts when owner is deactivated',
+                        help_text: t('admin.service.disableBotWhenOwnerIsDeactivated'),
+                        help_text_default: 'When a user is deactivated, disables all bot accounts managed by the user. To re-enable bot accounts, go to **Integrations > Bot Accounts**',
+                        isDisabled: it.stateEquals('ServiceSettings.EnableBotAccountCreation', false),
```

Comment:
While testing, I realized we should **not** disable the `ServiceSettings.DisableBotsWhenOwnerIsDeactivated` setting if bot accounts are disabled. **We should just always keep this setting editable.** This is because even if bot accounts creation is disabled, there may be existing bot accounts active in the system.

I'm sorry for this change! I realize you've implemented as specified in the help wanted issue.

Reply:
Thanks for the help :) That makes sense what you wrote. I'll get these changes in then.